dedupe/ecr_to_fhir_conversion/ECR to FHIR setup.ipynb

```diff
@@ -0,0 +1,196 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0d9247a0-f8e3-435b-932b-da2aef3fe9e2",
+   "metadata": {},
+   "source": [
+    "# Overview\n",
+    "\n",
+    "This notebook is intended is to walk you through the process of getting the ECR Viewer running locally with a fully populated database of dummy LA County data. There are several steps that need to be taken:\n",
+    "\n",
+    "1. Clone the PHDI repo\n",
+    "2. Download the LAC zip file\n",
+    "3. Convert all of the XML data into FHIR\n",
+    "4. Start looking for errors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4102572c-df5d-47dc-8119-fe7f71aae743",
+   "metadata": {},
+   "source": [
+    "### 1. Clone the PHDI repo\n",
+    "\n",
+    "If you've got SSH keys set with your GitHub account, then you can run `git@github.com:CDCgov/phdi.git`. Otherwise, you can run `https://github.com/CDCgov/phdi.git`. Either way, you should end up with a directory called `phdi`.\n",
+    "\n",
+    "**NOTE**: you should clean up the sample data that comes with the repo by deleting the data in the `phdi/containers/ecr-viewer/seed-scripts/baseECR` and `phdi/containers/ecr-viewer/seed-scripts/baseECR`."
```

Comment:
What is this sample data?

Reply:
The sample data is the data that already exists in the PHDI repo [here](https://github.com/CDCgov/phdi/tree/main/containers/ecr-viewer/seed-scripts/baseECR).